code_generator/GdkCodeGenerator/src/CodeGeneratorOptions.cs

```diff
@@ -34,6 +35,10 @@ public static CodeGeneratorOptions ParseArguments(ICollection<string> args)
                     "REQUIRED: the directory to output the network types and serialization code to",
                     u => options.NetworkTypesOutputDirectory = u
                 },
+                {
+                    "schema-compiler=", "REQUIRED: the schema compiler executable to use",
```

Comment:
Nit: make this `schema-compiler-path` and similar for all the variables.

Reply:
Done